docs/developer-guide/api.md

```diff
@@ -103,8 +106,117 @@ Response:
 }
 ```
 
+### GET /api/v2/action/listLinks
+The `listLinks` action responds with a list of URLs that have been shortened and that are associated with authenticated user.
+The list of URLs is paginated.
+
+Arguments:
+
+ - `per_page`: defines the count of lists per page. (Default is `15`)
+ - `page`: defines the page that should be displayed.
+ - `response-type`: `plain-text` or `json`. Whether the response will be a list in CSV or JSON format. 
+
+Example: GET `http://example.com/api/v2/action/listLinks?key=API_KEY_HERE&response_type=json&page=1&per_page=2`
+
+Response:
+```
+{
+    "action": "listLinks",
+    "result": {
+          "total": 4,
+          "per_page": 2,
+          "current_page": 1,
+          "last_page": 2,
+          "next_page_url": "http://example.com/api/v2/action/listLinks?page=2",
```

Comment:
Are the metadata attributes shown in this doc snippet implemented in this PR? I can try to implement those attributes myself if they're not yet implemented. I couldn't find any references to those attributes in the code.